src/solidity-frontend/solidity_grammar.h

```diff
@@ -303,8 +309,8 @@ enum ExpressionT
   // rule Tuple
   Tuple,
 
-  // rule revert
-  Revert,
+  // Mapping
```

Comment:
Why do we need this comment? You could expand it further instead.

Reply:
My mistake. It should be `rule Mapping`, something like:
```c++
enum ExpressionT{
  // rule identifier
  DeclRefExprClass,

  // rule literal
  Literal,

  // rule Tuple
  Tuple,

  // rule Mapping
  Mapping,
}
```

Basically, they are used to identify the type of the expression.